server/Arcadia.Assistant.CSP/Vacations/CspEmployeeVacationsRegistry.cs

```diff
@@ -6,28 +6,58 @@
     using System.Threading.Tasks;
 
     using Akka.Actor;
+    using Akka.Event;
 
     using Arcadia.Assistant.Calendar.Abstractions;
     using Arcadia.Assistant.Calendar.Abstractions.EmployeeVacations;
+    using Arcadia.Assistant.Calendar.Abstractions.EventBus;
     using Arcadia.Assistant.Calendar.Abstractions.Messages;
 
     public class CspEmployeeVacationsRegistry : UntypedActor, ILogReceive
     {
         private readonly VacationsSyncExecutor vacationsSyncExecutor;
         private readonly string employeeId;
 
+        private readonly ILoggingAdapter logger = Context.GetLogger();
+
         public CspEmployeeVacationsRegistry(
             VacationsSyncExecutor vacationsSyncExecutor,
             string employeeId)
         {
             this.vacationsSyncExecutor = vacationsSyncExecutor;
             this.employeeId = employeeId;
+
+            Context.System.Scheduler.ScheduleTellOnce(
```

Comment:
why not just Tell to self.

Reply:
I'm not sure I can just Tell in constructor.

Reply:
you can.

Reply:
Ok then, will fix it in next PR.